agent/util-scripts/pbench-register-tool

```diff
@@ -205,14 +215,16 @@ while true; do
 	esac
 done
 
-if [[ -z ${name} ]]; then
+# allow --test-args without --name
```

Comment:
Preceding a logic block which writes an error log and exits with `1`, the "allow" phrase here seems out of place.

Disallow missing `--name` without `--test-args`???